README.md

```diff
@@ -151,6 +151,76 @@ To stop Explorer after it has been started with `PM2`, issue the following comma
 
 `pm2 stop lisk-explorer`
 
+## Docker
+
+### Using docker-compose
+
+Update `docker-lisk-core.env` to choose your preferred node. You can easily switch between Mainnet and Testnet nodes by changing content of the env file.
+
+#### Starting application
+
+To start Explorer type the following command:
+
+```
+docker-compose up -d
+```
+
+It will use lastest available version from local hub by default.
+
+#### Stopping application
+
+The following command will remove all containers defined by the `docker-compose.yml`.
+
+```
+docker-compose down --volumes --rmi local
```

Comment:
Do we really want `--rmi`? If so we should document that not only "all containers" will be deleted.

Reply:
We could expand that section, I think it would be nice to clean up after the process stops.
The proposed param deletes untagged images only, a typical user won't need them anyway.